src/mjolnir/pbfgraphparser.cc

```diff
@@ -2387,10 +2387,16 @@ struct graph_callback : public OSMPBF::Callback {
         if (tokens.size() == 2 && tmp.size()) {
 
           uint16_t mode = 0;
-          if (boost::algorithm::starts_with(tag_.first, "motorcar:conditional") ||
-              boost::algorithm::starts_with(tag_.first, "motor_vehicle:conditional")) {
+          if (boost::algorithm::starts_with(tag_.first, "motor_vehicle:conditional")) {
             mode = (kAutoAccess | kTruckAccess | kEmergencyAccess | kTaxiAccess | kBusAccess |
                     kHOVAccess | kMopedAccess | kMotorcycleAccess);
+          } else if (boost::algorithm::starts_with(tag_.first, "motorcar:conditional")) {
+            if (type == kTimedAllowed) {
```

Comment:
the build fails @nvcnvn 
```suggestion
            if (type == AccessType::kTimedAllowed) {
```